### ISSUE TITLE
Color Contrast Issues with Tags

### ISSUE THREAD

**Issue by AdamJ** [dekkerglen/CubeCobra#2579]
**Describe the bug**
When selecting a tag color, the contrast between the background color and the text often makes it difficult to read what the card name is. 

**To Reproduce**
1. Open a card
2. Add a tag to the card
3. Click "Display -> Set Tags Colors"
4. Select a color from the available dropdown
5. Save & view the color with text

**Expected behavior**
- Whenever a tag color is selected that does not meet a11y accessibility guidelines, the text will invert to use a lighter color.
- Tag colors are preselected to provide proper contrast against the text in order to prevent the readability issues.

**Screenshots**
I have included screenshots demonstrating the problematic color combinations, excluding Orange and Yellow.

<img width="750" alt="Image" src="https://github.com/user-attachments/assets/d02e075e-f5e5-476e-8695-442d3f01d65b" />
<img width="744" alt="Image" src="https://github.com/user-attachments/assets/9cc8bd4c-f4a7-447e-b3ba-0b1f4f341b75" />
<img width="747" alt="Image" src="https://github.com/user-attachments/assets/1c9c2ba5-3730-490e-8eb1-51e95369ef3f" />
<img width="744" alt="Image" src="https://github.com/user-attachments/assets/dcdde51a-f131-450c-ab2d-a2a61dfd37c8" />

**Desktop (please complete the following information):**
 - OS: macOS 15.2
 - Browser Arc
 - Version 1.78.1

**Comment by KaelenProctor** [comment]
I'm taking a look at this using CSS contrast filter, so that it automatically increases contrast against the background

**Comment by AdamJ** [comment]
Thanks for taking care of this! Happy to help on any design work that may come up in the future.
